### ISSUE TITLE
Making adding new authors easier/more intuitive

### ISSUE THREAD

**Issue by evabenitoembo** [substance/texture#1140]
Entering authors is very cumbersome. Contextual 'insert' menu should have authors. Form should appear on page for First, Last name, email.

**Comment by michael** [comment]
Agreed this can be improved. As a first measure, would it be ok to enable Insert Author in the manuscript, but switch the context to the metadata view for editing?

**Comment by evabenitoembo** [comment]
Ok as a first measure, but we believe that eventually it should be possible to do it 'on the fly' in manuscript mode
